sp/src/game/client/client_ez2.vpc

```diff
@@ -36,9 +36,11 @@ $Project "Client (Entropy Zero 2)"
 		{
 			$File	"ez2\c_ez2_player.cpp"
 			$File	"ez2\c_ez2_player.h"
+			$File	"ez2\c_npc_assassin.cpp"
```

Comment:
This line should already be merged, right? Is this branch up to date with ez2/mapbase?

Reply:
Yes. This change just moves the line around to be in proper alphabetic order, which it mistakenly wasn't before.